plugins/woocommerce/tests/api-core-tests/tests/system-status/system-status-crud.test.js

```diff
@@ -30,44 +16,87 @@ test.describe( 'System Status API tests', () => {
-					wp_multisite: expect.any( Boolean ),
-					wp_memory_limit: expect.any( Number ),
-					wp_debug_mode: expect.any( Boolean ),
-					wp_cron: expect.any( Boolean ),
-					language: expect.any( String ),
-					external_object_cache: null,
-					server_info: expect.any( String ),
-					php_version: expect.any( String ),
-					php_post_max_size: expect.any( Number ),
-					php_max_execution_time: expect.any( Number ),
-					php_max_input_vars: expect.any( Number ),
-					curl_version: expect.any( String ),
-					suhosin_installed: expect.any( Boolean ),
-					max_upload_size: expect.any( Number ),
-					mysql_version: expect.any( String ),
-					mysql_version_string: expect.any( String ),
-					default_timezone: expect.any( String ),
-					fsockopen_or_curl_enabled: expect.any( Boolean ),
-					soapclient_enabled: expect.any( Boolean ),
-					domdocument_enabled: expect.any( Boolean ),
-					gzip_enabled: expect.any( Boolean ),
-					mbstring_enabled: expect.any( Boolean ),
-					remote_post_successful: expect.any( Boolean ),
-					remote_post_response: expect.any( String ),
-					remote_get_successful: expect.any( Boolean ),
-					remote_get_response: expect.any( String ),
-				} ),
-			} )
-		);
+		// local environment differs from external hosts.  Local listed first.
+		if ( ! shouldSkip ) {
+			expect( responseJSON ).toEqual(
+				expect.objectContaining( {
+					environment: expect.objectContaining( {
+						home_url: expect.any( String ),
+						site_url: expect.any( String ),
+						version: expect.any( String ),
+						log_directory: expect.any( String ),
+						log_directory_writable: expect.any( Boolean ),
+						wp_version: expect.any( String ),
+						wp_multisite: expect.any( Boolean ),
+						wp_memory_limit: expect.any( Number ),
+						wp_debug_mode: expect.any( Boolean ),
+						wp_cron: expect.any( Boolean ),
+						language: expect.any( String ),
+						external_object_cache: null,
+						server_info: expect.any( String ),
+						php_version: expect.any( String ),
+						php_post_max_size: expect.any( Number ),
+						php_max_execution_time: expect.any( Number ),
+						php_max_input_vars: expect.any( Number ),
+						curl_version: expect.any( String ),
+						suhosin_installed: expect.any( Boolean ),
+						max_upload_size: expect.any( Number ),
+						mysql_version: expect.any( String ),
+						mysql_version_string: expect.any( String ),
+						default_timezone: expect.any( String ),
+						fsockopen_or_curl_enabled: expect.any( Boolean ),
+						soapclient_enabled: expect.any( Boolean ),
+						domdocument_enabled: expect.any( Boolean ),
+						gzip_enabled: expect.any( Boolean ),
+						mbstring_enabled: expect.any( Boolean ),
+						remote_post_successful: expect.any( Boolean ),
+						remote_post_response: expect.any( String ),
+						remote_get_successful: expect.any( Boolean ),
+						remote_get_response: expect.any( String ),
+					} ),
+				} )
+			);
+		} else {
+			expect( responseJSON ).toEqual(
```

Comment:
There's a few differences, maybe due to different versions of PHP, or different web servers?  The `remote_get_` and `remote_post_` responses are returned as numbers from Atomic (which they are).  `wp-env` returns strings.  `external_object_cache` returns a `null` on `wp-env` and returns a Boolean value on Atomic.